src/Fixer/Basic/BracesFixer.php

```diff
@@ -364,7 +364,7 @@ function ($item) {
                     $nextLineCanBeIndented = false;
                     if ($nestToken->equalsAny(array(';', '}'))) {
                         $nextLineCanBeIndented = true;
-                    } elseif ($this->isCommentWithFixableIndendation($tokens, $nestIndex)) {
+                    } elseif ($this->isCommentWithFixableIndentation($tokens, $nestIndex)) {
```

Comment:
Nice catch! Even with this explicit diff I had to read it three times to spot what was wrong